### PR TITLE
Stop silently failing if logging is not configured

### PR DESCRIPTION
**Pull request by ketanyelp**

### CODE REVIEW

File: tests/config_test.py [hunk 1]
```diff
@@ -85,3 +85,18 @@ def test_configure_from_object_changes_scribe_disable(self):
...
+            (
+                sys.executable, '-c',
+                'import clog\n'
+                'try:\n'
+                '   clog.log_line("foo", "bar")\n'
+                'except Exception as e:\n'
+                '   print(e.__class__.__name__)\n'
+            ),
+            stdout=subprocess.PIPE,
+        )
```
Comment (asottile) [inline tests/config_test.py]: we don't support python2.6 any more, you can use `subprocess.check_output`

Comment (ketanyelp) [inline tests/config_test.py]: I'm not sure why the travis build is failing. It says the tox command can't be found. Any ideas?

**Comment by ketanyelp** [comment]
Got the tests to run by adding tox-travis to requirements.txt, but some tests are failing. They all pass locally though...

**Comment by ketanyelp** [comment]
Ok thanks. I reverted my attempted fixes of the build.

File: tests/config_test.py [hunk 2]
```diff
@@ -80,8 +80,17 @@ def test_configure_from_object_changes_scribe_disable(self):
...
                 'clog.config.configure_from_object(C)\n'
                 'print(clog.config.scribe_disable)\n'
-            ),
-            stdout=subprocess.PIPE,
-        )
-        out = proc.communicate()[0].decode('UTF-8')
+            )).decode('UTF-8')
         assert out == 'True\nFalse\n'
+
+    def test_logging_not_configured(self):
```
Comment (jwien001) [inline tests/config_test.py]: Can you additionally test that logging does work when configured via the 3 different methods?

Comment (ketanyelp) [inline tests/config_test.py]: Done.

**Comment by mp-e** [comment]
Ship it. @asottile can merge this I believe when he is ready.

**Comment by asottile** [comment]
:boat: thanks for going the extra mile on this :+1:

**Comment by asottile** [comment]
I'm going to revert this and then fix it, this is breaking downstream consumers who configure through staticconf and not explicitly

**Pull request merged**